.buildkite/pipeline.yml

```diff
@@ -1,4 +1,24 @@
-steps:
-  - label: "Placeholder Pipeline"
-    command: echo "Hello!"
+agents:
+  provider: "gcp"
+  machineType: "n1-standard-8"
+  useVault: true
```

Comment:
just checking: do we need this? not immediately seeing where we need a Vault token in this pipeline

Reply:
Good call - let's find out ;) incoming commit